packages/liveblocks-node/src/client.ts

```diff
@@ -84,6 +86,66 @@ type RoomInfoOriginal = Omit<RoomInfo, "lastConnectionAt" | "createdAt"> & {
   createdAt?: string;
 };
 
+type CommentReactionUser =
+  CommentDataOriginal["reactions"][number]["users"][number];
```

Comment:
Maybe expose this type as a building block in `@liveblocks/core`, and use it in the definition of `CommentDataOriginal`, instead of defining it as one big type and then "reaching into" that big type to get the smaller type out and alias it. Keeping types as small as possible generally really helps with composability and avoids having to write types like this.